ethereum_bridge/src/protocol/transactions/ethereum_events/events.rs

```diff
@@ -145,14 +147,14 @@ where
             // TODO: query denomination of the whitelisted token from storage,
             // and print this amount with the proper formatting; for now, use
             // NAM's formatting
-            if !asset_count.erc20_amount.is_zero() {
+            if asset_count.mint_erc20s() {
                 tracing::info!(
                     "Minted wrapped ERC20s - (asset - {asset}, receiver - \
                      {receiver}, amount - {})",
                     asset_count.erc20_amount.to_string_native(),
                 );
             }
-            if !asset_count.nut_amount.is_zero() {
+            if asset_count.mint_nuts() {
```

Comment:
I know Tomas already commented on this and you replied you'd handle it in a later PR, but I like how it was before.